persistence/azure-table/configuration_code_astp_[3,).partial.md

```diff
@@ -67,23 +67,21 @@ In case installers are enabled, but there's a need to opt out from creating the
 
 snippet: EnableInstallersConfigurationOptingOutFromTableCreation
 
-### Partitioning and compatibility helpers
+### Partitioning and compatibility mode helpers
 
-To atomically store data within the same transaction, all data written as part of the same message handling pipeline needs to share the same partition key.
-When storing sagas, saga IDs are by default deterministically derived from the saga data: the correlation property name and the correlation property value. Therefore, in conversations concerning a saga as part of the message handling pipeline, this deterministically derived ID is a perfect fit to be used as a partition key. Unfortunately, the saga ID assignment happens quite late during the message handling process and is not exposed to the user code.
+During a given message handling pipeline, multiple data storage operations may occur. To commit them atomically, in a single transaction, they must share a partition key. In conversations involving a saga, the saga ID is a good candidate for a partition key. Unfortunately, saga IDs are determined late in the message handling process and are not exposed to user code.
 
 This makes it difficult to:
 
-* Support transactional processing for existing sagas that were stored with the previous version of the persister and may have in-flight messages
```

Comment:
> and may have in-flight messages

This is redundant. An existing saga that was stored with the previous version of the persister will only be acted upon as a result of a message, and it is irrelevant whether or not there are _other_ "in-flight" messages.

Reply:
You can shut down and upgrade an endpoint at any time where scenarios like the following are possible:

- Shutdown during the existing inflight message for a saga and might not be handled and rolled back
- During shutdown messages that are destined to go to the saga instance are piling up in the queue to be processed after the endpoint starts
- Messages for the saga instance are already in the queue for the saga instance but not processed yet because the endpoint is shut down

These are the scenarios we summarized under in-flight.

Reply:
@danielmarbach those scenarios are all true, but how does "and may have in-flight messages" add any value to the sentence? The current version (in this PR) says:

> * Enable transactional saga data storage for existing sagas that were stored with the previous version of the persister.

Do we need more than that? This statement is true regardless of whether there are currently messages in the queue for the saga ("in-flight") or whether there will be messages sent to the saga in one year from now.

Reply:
Discussed with @danielmarbach and we agree that that the new bullet is OK.